src/accelerate/accelerator.py

```diff
@@ -17,10 +17,12 @@
 import sys
 import warnings
 from contextlib import contextmanager
+from pathlib import Path
 from typing import List, Optional, Union
 
 import torch
 
+from accelerate.accelerate.src.accelerate.tracking import TensorBoardTracker
```

Comment:
```suggestion
from .tracking import TensorBoardTracker
```
:scream:

Reply:
*That is what we call VSCode not being friendly and me missing it!*